recipe/meta.yaml

```diff
@@ -1,22 +1,20 @@
-{% set name = "gts" %}
-{% set version = "0.7.6" %}
-
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: gts
+  version: 0.7.6
```

Comment:
Is there a particular reason, why the (name and) version had to be made explicit here? Consider making at least the version a template variable.

Reply:
yeah, that looks to me not really required, even if this is more nit